### ISSUE TITLE
Redirect URL validation fails when URL has Query String params

### ISSUE THREAD

**Issue by ralmeida** [evonove/django-oauth-toolkit#211]
I was able to resolve the issue in the method  in ```oauth2_validators.py line: 271```
```def validate_redirect_uri(self, client_id, redirect_uri, request, *args, **kwargs):``` 

by changing the following from 
```python
def validate_redirect_uri(self, client_id, redirect_uri, request, *args, **kwargs):
    return request.client.redirect_uri_allowed(redirect_uri)
```
to 
```python
def validate_redirect_uri(self, client_id, redirect_uri, request, *args, **kwargs):
    baseUrl = redirect_uri
    if(redirect_uri.find('?') > -1):
        baseUrl = redirect_uri[:redirect_uri.find('?')]
    return request.client.redirect_uri_allowed(baseUrl)
```

It would be really helpful if this made it into the actual codebase. 

As I didn't see any guidelines on contributions, I figured I'd add it here.

Cheers,
Ron

**Comment by masci** [comment]
Fixed in #239

**Comment by SilviyaRepo** [comment]
Hi..How could you edit the code in oauth2_provider.. Can you guide me ? I am new to django.. I want to restrict multiple access token insertion in database for the same user if he/she has unexpired token in table,,, Currently it adds new token in DB..How can I achieve this?
